apps/cli/src/commands/list.ts

```diff
@@ -26,20 +27,22 @@ export const handleListNamesCommand = async (
   }
 
   let prettified = configObjects.map(
-    ({ name, verified: _, tags: tagsArray, engine, author }) => {
+    ({ name, verified: _, tags: tagsArray, engine, author, slug }) => {
       const tags = tagsArray.join(", ");
 
       if (search && (name === search || tagsArray.includes(search))) {
         return {
-          name: chalk.bold.cyan(name),
+          name: chalk.bold.cyan(
+            terminalLink(name, `https://codemod.com/registry/${slug}`),
+          ),
```

Comment:
i'd suggest moving terminalLink(...) to a separate variable and re-using it on lines 36 & 45